ragna/assistants/_bedrock.py

```diff
@@ -0,0 +1,79 @@
+from typing import cast
+import boto3
+import json
+from ragna.core import RagnaException, Source
+
+from ._api import ApiAssistant
+
+
+class AmazonBedrockAssistant(ApiAssistant):
+    _API_KEY_ENV_VAR = "AWS_ACCESS_KEY_ID"
```

Comment:
Although we set it here, it is never used anywhere. That hints to the fact that `ApiAssistant` is probably not the right abstraction and you are better off subclassing from `ragna.core.Assistant` directly.